### ISSUE TITLE
A11y / Sensei / Learn - The lesson status icon has no accessible name

### ISSUE THREAD

**Issue by joedolson** [WordPress/Learn#2780]
## Type of feedback
//dev

## Description

The lesson status icon in a list of lessons does not have any accessible name to communicate to screen reader users which lessons they have completed. The icon is visual only.

Each possible icon should have an accessible name so that all users are able to discern which lessons are completed.

In the screenshot provided, there is no distinction between the status of the two lessons for screen reader users.

## Screenshots or screen recording (optional)

![image](https://github.com/user-attachments/assets/fed096ee-8f23-4439-97a4-8cb4c4cd6fbe)

**Comment by joedolson** [comment]
Also true in the course navigation on single lessons:

![image](https://github.com/user-attachments/assets/471da223-932c-4120-a676-d6f8d5bfa29f)
